Cargo.toml

```diff
@@ -25,6 +25,7 @@ members = [
    "packages/fuel-indexer-types",
    "plugins/forc-index",
    "plugins/forc-postgres",
+   "plugins/plugin-tests",
```

Comment:
Could we maybe name this to `forc-index-tests` for consistency?
- `fuel-indexer` --> `fuel-indexer-tests`
- `forc-index` --> `forc-index-tests`